CHANGELOG.md

```diff
@@ -6,12 +6,15 @@
 
 ### Changed
 
+- Fix opacity with photometric interpretations.
+
 ## 0.8.1
 
 ### Added
 
 - Property `onViewStateChange` of all viewers accepts a callback for deck.gl view state changes.
 - Add a click handler to the overview that centers the detail view on the click position. The handler can be turned off by setting the `PictureInPictureViewer` property `clickCenter` to `false`.
+- Support interleaved RGB OME-TIFF files.
```

Comment:
Why did this get moved?

Reply:
It was in the wrong version, as I painfully found out when handing Vitessce off to Heath and RGB interleaved images didn't work.